### ISSUE TITLE
a rule run multiple times

### ISSUE THREAD

**Issue by BlankRain** [CoNarrative/precept#109]
Hi, I have two rules added  to todomvc .

```

(rule print-one-when-a-pressed
  {:group :action}
  [[_ :input/key-code 65]]
  =>
  (.log js/console 1))  

(rule print-two-when-a-pressed
  {:group :action}
  [[_ :input/key-code 65]]
  =>
  (.log js/console 2))



```

when I press a .and I got this ..

![image](https://user-images.githubusercontent.com/10811573/33249273-a2091db6-d364-11e7-96e9-0515251513a0.png)

why I have 4 times "2" output and 1 times "1" output...

**Comment by alex-dixon** [comment]
I performed the following and could not reproduce:

`lein new precept my-proj`
` cd my-proj`
`lein figwheel`
Opened browser and saw "Hi / there!"
Copied and pasted rules into top of `rules.cljs`
Saved the file
entered "a" but saw no change, so refreshed the page
Console output was as expected => 2 1

Based on this I'd recommend stopping figwheel and doing `lein do clean, figwheel` if you haven't already and seeing if this fixes the issue.

If you're running the example from a clone of the latest master I should probably test that version instead of the template version. It's possible the library version used in the template is out of sync with the latest, which might also explain why I'm not getting hot reloading (we added this as a feature a while back).
